Makefile

```diff
@@ -52,3 +51,22 @@ image:
 	$(MAKE) -C app-policy image IMAGETAG=$(GIT_VERSION) VALIDARCHES=$(ARCH)
 	$(MAKE) -C typha image IMAGETAG=$(GIT_VERSION) VALIDARCHES=$(ARCH)
 	$(MAKE) -C node image IMAGETAG=$(GIT_VERSION) VALIDARCHES=$(ARCH)
+
+###############################################################################
+# Release logic below
+###############################################################################
+# Build the release tool.
+hack/release/release: $(shell find ./hack/release -type f -name '*.go')
+	$(DOCKER_RUN) $(CALICO_BUILD) go build -v -o $@ ./hack/release/release.go
+
+# Install ghr for publishing to github.
+hack/release/ghr: 
+	$(DOCKER_RUN) -e GOBIN=/go/src/$(PACKAGE_NAME)/hack/release/ $(CALICO_BUILD) go install github.com/tcnksm/ghr
```

Comment:
Suggest putting this in go-build or at least pinning to a particular commit ID. I think go install supports that now?

Reply:
Yeah, that's probably a good idea. I was planning on looking into just importing this (since we're all in golang now) - I might investigate that a bit more first but if that's not trivial I'll switch to pinning.